dotnet-install.cmd

```diff
@@ -0,0 +1,45 @@
+@echo off
+@if defined _echo echo on
+
+:install_dotnet_cli
+setlocal
+  set DotNet_Path=%~dp0tools\bin
+  set Init_Tools_Log=%DotNet_Path%\install.log
+  set /p DotNet_Version=<"%~dp0DotNetCLIVersion.txt"
+  set DotNet_Installer_Url=https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.ps1
+
+  REM dotnet.exe might exist, but it might not be the right version.
+  REM Here we verify that if it is not the right version, then we install it
+  if exist "%DotNet%" (
```

Comment:
I'm guessing `%DotNet%` has to be defined by the caller? Could we define it here to make this script less prone to break?